### ISSUE TITLE
0.3.11 won't accept array for source

### ISSUE THREAD

**Issue by grayghostvisuals** [zont/gulp-usemin#104]
My project Transformicons uses this setup below in our gulpfile, but currently this setup breaks in 0.3.11 (I had to revert to 0.3.8) meaning only the ``index.html`` file is filtered through all the tasks while the rest of the files remain untouched.

```javascript
gulp.task('usemin', ['assemble', 'cssmin'], function() {
  var stream = gulp.src([
        paths.site + '/index.html',
        paths.site + '/builder.html',
        paths.site + '/docs.html'
      ])
      .pipe($.usemin({
        html: [$.minifyHtml({ empty: true })],
        css: [$.rev()],
        js: [$.uglify(), $.rev()]
      }))
      .pipe(gulp.dest(paths.site));

  return stream;
});
```

**Comment by grayghostvisuals** [comment]
@simplydenis Is there a better way to write this now based on the new updates?

**Comment by codefucker** [comment]
I have the same issue.

**Comment by pizthewiz** [comment]
Is this another report of #91?

**Comment by zont** [comment]
Fixed in 0.3.14
